src/app.rs

```diff
@@ -491,3 +502,24 @@ lazy_static! {
 fn validate_number(s: String) -> Result<(), String> {
     s.parse::<usize>().map(|_|()).map_err(|err| err.to_string())
 }
+
+fn validate_max_filesize(s: String) -> Result<(), String> {
+    let re = Regex::new(r#"^(\d+)([KMG])?$"#).unwrap();
```

Comment:
`\d` is Unicode aware, which includes a lot more than `[0-9]`, so I think you want to change this to `[0-9]`.